tests/system/action/user/test_create.py

```diff
@@ -290,68 +346,236 @@ def test_create_permission_superadmin(self) -> None:
+                "comment_$": {"1": "comment zu meeting/1"},
+                "vote_delegations_$_from_ids": {"1": [5, 6]},
+                "group_$_ids": {"1": [1]},
             },
         )
         self.assert_status_code(response, 200)
         self.assert_model_exists(
-            "user/3",
+            "user/7",
             {
-                "group_$_ids": ["1", "4"],
-                "group_$1_ids": [1],
-                "group_$4_ids": [5],
-                "username": "username",
+                "username": "username7",
+                "number_$": ["1"],
+                "number_$1": "number1",
+                "structure_level_$": ["1"],
+                "structure_level_$1": "structure_level 1",
+                "vote_weight_$": ["1"],
+                "vote_weight_$1": "12.002345",
+                "about_me_$": ["1"],
+                "about_me_$1": "about me 1",
+                "comment_$": ["1"],
+                "comment_$1": "comment zu meeting/1",
+                "vote_delegations_$_from_ids": ["1"],
+                "vote_delegations_$1_from_ids": [5, 6],
+                "meeting_ids": [1],
             },
         )
 
-    def test_create_permission_committee_manager_no_permission(self) -> None:
-        """ vote_weight_$ is in group B and may only work with meeting-Permission  """
+    def test_create_permission_group_B_user_can_manage_no_permission(self) -> None:
+        """ Group B fields needs explicit user.can_manage permission for meeting """
+        self.permission_setup()
+        self.set_organisation_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+        self.set_user_groups(self.user_id, [3])  # Empty group of meeting/1
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "usersname",
+                "number_$": {"1": "number1"},
+                "group_$_ids": {"1": [1]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You are not allowed to perform action user.create. Missing permission: Permission user.can_manage in meeting 1",
+            response.json["message"],
+        )
+
+    def test_create_permission_group_C_oml_manager(self) -> None:
+        """ May create group C group_$_ids by OML permission """
+        self.permission_setup()
+        self.set_organisation_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "usersname",
+                "group_$_ids": {1: [1]},
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/3",
+            {"group_$_ids": ["1"], "group_$1_ids": [1], "username": "usersname"},
+        )
+
+    def test_create_permission_group_C_committee_manager(self) -> None:
```

Comment:
please write a similar success test trying to add the user to the committee instead of the meeting.

Reply:
see test_create_permission_group_D_permission_with_OML